test/system/maintenance_tasks/runs_test.rb

```diff
@@ -162,7 +162,6 @@ class RunsTest < ApplicationSystemTestCase
 
       assert_text "Cancelling…"
       refute_button "Cancel"
-
```

Comment:
Nit: I think this was probably removed accidentally, but let's keep this line between the assertions and the `travel` line.